Microsoft.Data.Analysis.Interactive/Microsoft.Data.Analysis.Interactive.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
This should be removed. In the end, we will only want a single nuget package that contains both `Microsoft.Data.Analysis` and `Microsoft.Data.Analysis.Interactive` libraries.

Reply:
indeed

Reply:
if there are 2 dlls it makes it easy to load the Microsoft.Data.Analysis and not the extension one when loading the package. We have sample for that packaging